src/c-writer.cc

```diff
@@ -473,7 +472,6 @@ class CWriter {
   StackVarSymbolMap stack_var_sym_map_;
   SymbolSet global_syms_;
   SymbolSet local_syms_;
-  SymbolSet import_syms_;
```

Comment:
Was this simply unused?

Reply:
It was used for finding whether a symbol was imported or not, but it was redundant to `import_module_sym_map_`. This PR creates the `IsImport` method which checks if the name exists in `import_module_sym_map_`, and then we can get rid of `import_syms_`. I'll split this out along with the above.